src/stores/room-list/algorithms/tag-sorting/RecentAlgorithm.ts

```diff
@@ -25,7 +25,7 @@ import * as Unread from "../../../../Unread";
  * useful to the user.
  */
 export class RecentAlgorithm implements IAlgorithm {
-    public async sortRooms(rooms: Room[], tagId: TagID): Promise<Room[]> {
+    public async sortRooms(rooms: Room[], _: TagID): Promise<Room[]> {
```

Comment:
no thanks, let's keep the name even if unused.

Reply:
Ah, I fixed this in the rules but forgot to revert it. Thanks